app/config/runtimes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Utopia\App;
+use Appwrite\Runtimes\Runtimes;
+
+/**
+ * List of Appwrite Cloud Functions supported runtimes
+ */
+$runtimes = new Runtimes();
+
+$allowList = empty(App::getEnv('_APP_FUNCTIONS_ENVS')) ? false : \explode(',', App::getEnv('_APP_FUNCTIONS_ENVS'));
+
+$runtimes = array_filter($runtimes->getAll(), function ($key) use ($allowList) {
+    $isAllowed = $allowList && in_array($key, $allowList);
+
+    return $allowList ? $isAllowed : true;
+}, ARRAY_FILTER_USE_BOTH);
```

Comment:
I think we can consider moving most of the logic to the `Runtimes` class as a "filter or a "allowedList" feature